integration/test_aggregation_integration.py

```diff
@@ -13,8 +13,9 @@
 
 
 @pytest.mark.parametrize('partitioned_by_key', [True, False])
-def test_aggregate_and_query_with_different_windows(setup_teardown_test, partitioned_by_key):
-    table = Table(setup_teardown_test, V3ioDriver(), partitioned_by_key=partitioned_by_key)
+@pytest.mark.parametrize('flush_interval', [0, 1000])
```

Comment:
We need tests for `flush_interval` specifically. i.e. ones that would fail if the feature wasn't working. Something like a flow without WriteToTable followed by another flow reading the persisted data back (without reusing the Table object which would cache the data in memory).